src/test/java/nextstep/ladder/UserTest.java

```diff
@@ -0,0 +1,18 @@
+package nextstep.ladder;
+
+import nextstep.ladder.entity.User;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+public class UserTest {
+    @Test
+    @DisplayName("이름 길이 제한")
+    public void nameLength(){
+        assertThatIllegalArgumentException()
+                .isThrownBy(() ->
+                        new User("nameLengthOver")
+                );
```

Comment:
아래처럼 메시지와 함께 테스트 해 수도 있겠네요.

```suggestion
        assertThatIllegalArgumentException()
                .isThrownBy(() ->
                        new User("nameLengthOver")
                ).withMessageMatching("이름은 5자를 초과 할 수 없습니다.");
```